### PR TITLE
Use bandwidth checking same as other platforms.

### PR DESCRIPTION
**Pull request by jigar-f**

### CODE REVIEW

File: lib/features/vpn/vpn_bandwidth.dart [hunk 1]
```diff
@@ -8,13 +8,8 @@ class VPNBandwidth extends StatelessWidget {
   Widget build(BuildContext context) {
     return sessionModel
         .bandwidth((BuildContext context, Bandwidth bandwidth, Widget? child) {
-      // User does not have bandwidth cap off
-      if (bandwidth.remaining > 0 || isDesktop()) {
```
Comment (jigar-f) [inline lib/features/vpn/vpn_bandwidth.dart:12]: There was an issue here, If use have not used any of the data here then it shows the progress bar as complete.

File: desktop/datacap/datacap.go [hunk 2]
```diff
@@ -92,6 +99,27 @@ func (dc *dataCap) processQuota(out chan<- interface{}, quota *bandwidth.Quota)
 		})
 	}
 }
+func getBandwidth(quota *bandwidth.Quota) (int, int, int) {
```
Comment (jigar-f) [inline desktop/datacap/datacap.go:102]: @atavism I think should use what we are using on other platforms just for consistency and we can avoid some platform conditions on the flutter end.

**Comment by jigar-f** [comment]
I am merging this, Let me know if you want me to revert this chanegs.

**Comment by atavism** [comment]
OK, now that these changes were reverted, "When first installed, I saw a data cap, but when I did a clean install on another Windows user account, I saw no data cap. After about 10 minutes of use, the data cap showed up again." is an issue again.

**Comment by jigar-f** [comment]
@atavism In this case, I think there might be some issue with the bandwidth check on the desktop, Could you see why there lag on that

**Pull request merged**